components/gantry/oneaxis/oneaxis.go

```diff
@@ -284,10 +246,9 @@ func (g *oneAxis) homeEncoder(ctx context.Context) error {
 	return nil
 }
 
-func (g *oneAxis) linearToRotational(positions float64) float64 {
-	theRange := g.positionLimits[1] - g.positionLimits[0]
+func (g *oneAxis) rotationalToLinear(positions float64) float64 {
```

Comment:
Oh, I get it. You've renamed the function. but I suspect the old name was more accurate: the first thing we do with the argument is divide it by a length in millimeters to get a unitless quantity, which makes me suspect the input is linear and the output is rotational. So, I'd change the name back.

but this looks very intentional by you, which makes me wonder if I've missed something. Why did you change the name?

Reply:
yes you did, positions are the stepper ticks reported by the stepper in ticks travelled, an absolute rotational position,  that we are converting to a linear range knowing the position angle - mm ratio. 
I can call it `mapMotorToAxisPosition` if it makes it clearer.

Reply:
Or open to new names.

Reply:
@penguinland renamed it - let me know what you think of this new one.

Reply:
Yup, I like the new new name! Thanks